workflows/templates/biascorrectdownscale.yaml

```diff
@@ -404,15 +421,24 @@ spec:
                   value: 10
                 - name: lon-chunk
                   value: 180
-          - name: validate-biascorrected
+          - name: apply-non-polar-dtr-ceiling-on-biascorrected
+            template: apply-non-polar-dtr-ceiling
             depends: rechunk-biascorrected
+            arguments:
+              parameters:
+                - name: in-zarr
+                  value: "{{ tasks.rechunk-biascorrected.outputs.parameters.out-zarr }}"
+                - name: out-zarr
+                  value: "{{ tasks.rechunk-biascorrected.outputs.parameters.out-zarr }}" # writing to input file !
```

Comment:
I don't thiiiink this will work... I suspect it's going to need to write to a new output file.